models/app.model.js

```diff
@@ -18,4 +18,30 @@ function getArticleById (chosenId) {
     })
 }
 
-module.exports = {getAllTopics, getArticleById}
\ No newline at end of file
+function getAllArticlesModel () {
+    return db.query(`SELECT
+    articles.author,
+    title,
+    articles.article_id,
+    topic,
+    articles.created_at,
+    articles.votes,
+    article_img_url,
+    COUNT (comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY
+    articles.author,
+    title,
+    articles.article_id,
+    topic,
+    articles.created_at,
+    articles.votes,
+    article_img_url
+    ORDER BY created_at DESC`)
+    .then(data => {
+        return data.rows;
+    })
+}
```

Comment:
Excellent! You've implemented the comment_count using an aggregate function which is a great! and done the sort by and order by, good work.